<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leaf Area Measurement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBoxShowRecursive">
      <property name="text">
       <string>List also files from subdirectories</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Analyse all files in list view</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>16</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionParameters"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionParameters">
   <property name="text">
    <string>&amp;Parameters</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>treeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShowRecursive</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>toggleRecursive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>699</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>analyseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>677</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>populateTreeView()</slot>
  <slot>selectFolder()</slot>
  <slot>toggleRecursive()</slot>
  <slot>selectImage()</slot>
  <slot>showOptionsDlg()</slot>
  <slot>analyseAll()</slot>
 </slots>
</ui>
